packages/block-library/src/media-text/index.js

```diff
@@ -156,16 +162,22 @@ export const settings = {
 			mediaUrl,
 			mediaWidth,
 			mediaId,
+			imageFill,
+			focalPoint,
 		} = attributes;
+		const imageFillStyle = imageFillStyles( mediaUrl, focalPoint );
```

Comment:
Minor: can we use imageFillStyles( mediaUrl, focalPoint ) inline to avoid a wasted computation if we are not in fill mode?